config.go

```diff
@@ -25,6 +29,15 @@ func LoadConfigFromPath(path string) (Config, error) {
 	if dsn, found := os.LookupEnv("TIGERBLOOD_DSN"); found {
 		config.DatabaseDsn = dsn
 	}
+	if dbMaxOpenConns, found := os.LookupEnv("TIGERBLOOD_DB_MAX_OPEN_CONNS"); found {
+		dbMaxOpenConns, err := strconv.Atoi(dbMaxOpenConns)
+		if err != nil {
+			config.DatabaseMaxOpenConns = dbMaxOpenConns
```

Comment:
Looks like this logic is wrong (if error then set to the errored value).

I also think it should be a fatal error if that environment variable is set, but fails to parse.